src/reducers/CompareResults.ts

```diff
@@ -24,6 +26,31 @@ const compareResults = createSlice({
       state.data = comparisonResults;
     },
   },
+  extraReducers: (builder) => {
+    builder
+      .addCase(fetchCompareResults.fulfilled, (state, action) => {
```

Comment:
### Optional
Here are some of my observations, totally optional, if fixed can improve performance and load time.

- Due to some reason, the calls to the API endpoint are duplicated. For example, if there are 3 new revisions selected there are 6 calls to the API endpoint where there should be only 3.

- (Related to duplication of API calls) ResultsView appears to render twice initially triggering duplicated calls of the useEffect() that contains a call to `dispatchFetchCompareResults`. If we can figure out why ResultsView is rendering twice, we can solve the problem of duplicated API calls.

Reply:
It only happens locally because of [StrictMode](https://stackoverflow.com/questions/61254372/my-react-component-is-rendering-twice-because-of-strict-mode), but not on prod.